src_new/constants.js

```diff
@@ -163,3 +163,8 @@ exports.HOOKS = {
 	"BID_RECEIVED": "HookForBidReceived",
 	"POST_AUCTION_KEY_VALUES": "HookForPostAuctionKeyValues"
 };
+
+exports.BIDDER_NAMES = {
+	"RUBICON" : "rubicon",
```

Comment:
try to keep a list of this feature enabled adapters in constants.js

Reply:
Done